Dockerfile

```diff
@@ -26,21 +49,19 @@ RUN apk --update add --no-cache --virtual build-dependencies gcc git curl py3-pi
     && addgroup spiderfoot \
     && adduser -G spiderfoot -h /home/spiderfoot -s /sbin/nologin \
                -g "SpiderFoot User" -D spiderfoot \
-    && rmdir /home/spiderfoot \
-    && cd /home \
-    && curl -sSL https://github.com/smicallef/spiderfoot/archive/v3.0.tar.gz \
```

Comment:
Was this intentionally removed? Where will the actual SpiderFoot installation be if it's not downloaded here?

Reply:
It's installing the files from the git workspace (see the "Copy . . " directive). This is a typical Docker workflow. If someone wants a v3.0 Docker image they checkout the v3.0 tag and build it. If they want an image with the latest changes, they checkout master and build that. For example, my spiderfoot fork is automatically building each commit using github actions and pushing them to Docker Hub with a tag based on the commit timestamp and git SHA.

Reply:
Makes sense, thanks!